### PR TITLE
fix: hologram material on pooled GLTFs, re-enable hologram material

### PR DESCRIPTION
**Pull request by pravusjif**
https://play.decentraland.zone/?renderer-branch=fix/hologramMaterialOnPooledAssets&position=-118,125

### CODE REVIEW

File: unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs
Comment (AjimenezDCL) [inline unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs:318]: Great job! I would wait for Brian's approval to understand the reasoning behind the the bailout, but looks good to me.

**Review by AjimenezDCL — APPROVED** [review]

File: unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs
Comment (pravusjif) [reply unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs:318]: As talked with Brian in the meeting with the 3 of us, I'm moving forward with merging this PR.

**Pull request merged**
